content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,278 @@
+  ```
+
+
+15. Use the command below to create the content bundle. The command uses the following flags:
+
+  |**Command Flag**|**Value**|
+  |---|---|
+  |`--api-key`|Spectro Cloud API key|
+  |`--project-id`|Palette project ID|
+  |`--cluster-profile-ids`|Cluster profile IDs. If you want to include multiple cluster profiles in the content bundle, add multiple cluster profile IDs separated by a comma.|
+  |`--palette-endpoint`|Palette API endpoint. The default Palette API endpoint is `api.spectrocloud.com`|
+  |`--outfile`|Path to write the final content bundle. |
+  You can issue `palette-edge build --help` to know about other available flags. 
+<br />
+
+  ```bash
+  palette-edge build --api-key $API_KEY \
+    --project-id $PROJECT_ID \
+    --cluster-profile-ids $PROFILE_ID \
+    --palette-endpoint api.spectrocloud.com \
+    --outfile content
+  ```
+  
+
+
+16. Use the command below to list all files in the current directory to verify that you created the content bundle successfully. The content bundle will have the following naming convention, `content-[randon-string]`, for example, **content-8e61a9e5**. 
+<br />
+
+  ```bash
+  ls -al
+  ```
+
+
+17. List the files in the content bundle folder using the following command. Replace the `content-[randon-string]` with the content bundle in your current directory. The output will display the compressed files for the core and the app content. 
+<br />
+
+  ```bash
+  ls -l content-[randon-string]
+  ```
+  ```bash hideClipboard
+  # Sample output   
+  total 3981104
+  -rw-rw-r-- 1 jb jb 1598552722 Jul 26 18:20 app-content-8e61a9e5.zst
+  -rw-rw-r-- 1 jb jb 2478086360 Jul 26 18:20 core-content-8e61a9e5.zst
+  ```
+
+
+18. Issue the following command to execute the **earthly.sh** script with elevated privileges. The `+iso` option specifies the build target. This command will generate an ISO image from the content bundle and other configurations you have specified in the **.arg** and **user-data** files. 
+<br />
+
+  ```bash
+  sudo ./earthly.sh +iso
+  ```
+  This command may take up to 15-20 minutes to finish depending on the resources of the host machine. 
+
+
+# Validate
+
+List the Edge installer ISO and checksum by issuing the following command from the **CanvOS/** directory.
+<br />
+
+```shell
+ls build/
+```
+
+```shell hideClipboard
+# Output
+palette-edge-installer.iso      
+palette-edge-installer.iso.sha256
+```
+<br />
+
+To validate, you can prepare an edge device using the Edge installer ISO by following the steps below:
+<br />
+
+1. Create a bootable USB flash drive using any third-party software. Most software that creates a bootable USB drive will validate the ISO image.
+
+
+2. Select a physical or virtual host machine to emulate as an edge device. Enable (Dynamic Host Configuration Protocol) DHCP on the edge device before proceeding with the installation process. Enabling DHCP is necessary for the device to obtain an IP address automatically from the network. 
```

Comment:
```suggestion
2. Select a physical or virtual host machine to emulate as an edge device. Enable (Dynamic Host Configuration Protocol) DHCP on the host before proceeding with the installation process. Enabling DHCP is necessary for the device to obtain an IP address automatically from the network. 
```
I think host is the right word here in case they go with VMware to validate the installer.